14/umbraco-commerce/tutorials/getting-started-with-commerce.md

```diff
@@ -0,0 +1,183 @@
+---
+description: "A guide to setting up Umbraco Commerce, including an introduction to basic concepts and configuration."
+---
+# A Beginner's Guide to Umbraco Commerce
+
+This tutorial is based and tested on the latest version of **Umbraco CMS version 14** and **Umbraco Commerce version 14**.
+
+## Introduction
+
+Umbraco Commerce (formerly known as Vendr) is an eCommerce platform that integrates with Umbraco CMS. It offers features such as product management, order processing, and payment integrations. This guide will walk you through setting up Umbraco Commerce.
+
+## Prerequisites
+
+* Visual Studio Code or any preferred IDE
+* SQL Database (LocalDB or any SQL server)
+* [Umraco CMS Requirements](../../umbraco-cms/fundamentals/setup/requirements.md)
+* [Umbraco CMS Installation](../../umbraco-cms/fundamentals/setup/install/README.md)
+* [Umbraco Commerce package](../getting-started/install.md)
+
+## Setting Up an Umbraco Project
+
+To set up an Umbraco project, see the [Installation](../../umbraco-cms/fundamentals/setup/install/README.md) article.
+
+## Installing Umbraco Commerce
+
+After your Umbraco site is set up, you need to [install the Umbraco Commerce package](../getting-started/install.md).
+
+If you have installed Umbraco CMS with an SQLite database, you might encounter issues or errors after installing Umbraco Commerce.
+
+![SQLite Error](images/sqlite-error.png)
+
+To resolve these issues, follow the steps mentioned in the [Configure SQLite support](../how-to-guides/configure-sqlite-support.md) article.
+
+## Configuring Umbraco Commerce
+
+To gain access to the **Commerce** section, additional configuration is needed. For more information, see the [Configuration](../getting-started/umbraco-configuration.md) article.
+
+## Accessing the Umbraco Backoffice
+
+Log in to the Umbraco backoffice by adding `/umbraco` at the end of your website URL. For example: `https://mywebsite.com/umbraco`.
+
+## Setting Up a Store
+
+1. Navigate to the **Settings** section.
+2. Click **+** next to **Stores**.
+3. Enter a **Name** for the Store. For this tutorial, I'll call it *Products*.
+
+    ![Create Store](images/create-store.png)
+4. Click **Create**.
+5. Configure the **General**, **Notification**, **Order**, **Product**, and **Gift Card Settings** in the **Details** tab.
+6. Select the **User Roles** and **Users** that must have access to manage the Store in the **Permissions** tab.
+
+    ![Store Permissions](images/store-permissions.png)
+7. Click **Save**.
+
+### Setting up a Location
+
+1. Navigate to the **Settings** section.
```

Comment:
Aren't we already in the **Settings** section based on the section above?

Reply:
Good point 😀